.circleci/config.yml

```diff
@@ -946,6 +945,20 @@ jobs:
           command: |
             /infrastructure/scripts/check_release_artifacts.sh ${CIRCLE_TAG}
 
+  build_uml_diagrams:
+    executor: builder_container_stable
+    <<: *container_config
+    steps:
+      - checkout
+      - run:
+          name: Install PlantUML dependencies
+          command: |
+            sudo apt-get -qq -y update && sudo apt-get -qq -y install graphviz librsvg2-bin
```

Comment:
@tolbrino @dincho Shall this rather go in https://github.com/aeternity/docker-builder/blob/d15fa8e500d4ec4ee808aac6c3ea89a5b42c8f22/Dockerfile ?

Reply:
(Eventually...)

Reply:
As per @dincho 's suggestion I used another pre-built image, which works fine.